app/views/application/_account-navigation.html.erb

```diff
@@ -0,0 +1,30 @@
+<% page_is ||= nil %>
+
+<nav aria-label="Account management">
+  <ul class="accounts-menu">
+    <li class="accounts-menu__item <%= "accounts-menu__item--current" if page_is == "your-account" %>">
```

Comment:
Not for this PR, but it should be possible to output these menu items using a loop and some kind of data object. Probably only worth implementing if there'll be more than three of them in future though, I guess.